### DISCUSSION TITLE
Patterns from multiple languages

### DISCUSSION THREAD

**Discussion by LiangWei88** [danielmiessler/Fabric#951] [Q&A]
Hello, I am seeking a tool to effectively manage prompts. I recently discovered Fabric and it appears to be exactly what I need. However, my native language is Chinese, and in my everyday usage I predominantly use Chinese prompt terms. How does your project handle submitting different languages' patterns? Or should users like me fork the repository for translation of our own unique needs?

**Comment by xssdoctor** [comment]
It should handle chinese just as well as the LLM does. If you want the output to be in chinese, I recommend creating a context called chinese and say in it "please respond only in chinese". if you use the --context flag (with the specified context), it will work. I've never tried in chinese, but this is what works for spanish

**Comment by LiangWei88** [comment]
Thank you. I happen to need this functionality right now. 
However, what I meant above is not the output in Chinese itself but that I plan to use language models trained on Chinese data for my project. Therefore, prompt terms should be in Chinese or a "Chinese-like" format. How do we manage different languages’ prompts? For example, how can we handle prompts in Chinese, French, Japanese etc.?

**Reply by lobodecrin-studio** [reply]
I love Fabric, and initially, I made my custom patterns so that the responses would always be in Spanish. But I wanted to use all the patterns that already exist in the repository, not just the custom ones.

So I had this idea: I generated a custom model with Ollama's Modelfile, using any LLM like llama3.1 or any other, and forced in that Modelfile that all responses are always in Spanish, regardless of whether the prompts from the patterns are written in English or Spanish.
Now I'm using my custom ollama model in fabric with the original patterns and the results are in spanish, even if the input content and the patterns are in english
